client/src/components/Login.js

```diff
@@ -3,39 +3,56 @@ import { Link, useNavigate } from "react-router-dom";
 import axios from 'axios';
 import { useUser } from "../contexts/UserProvider";
 import { BiArrowBack } from "react-icons/bi";
+import Alert from 'react-bootstrap/Alert';
+
 
 function Login() {
 	const [credentials, setCredentials] = useState({ email: "", password: "" });
-	const navigate = useNavigate();
+	const [show, setShow] = useState(false);
```

Comment:
Be more verbose. Specify what do you want to "show".
Instead of `show` and `setShow` do something like `showAlert` or `showLoginErrorAlert`.

Reply:
okay